### PR TITLE
Rename Folders to Axe.Windows.* instead of AccessibilityInsights.*

### PR DESCRIPTION
**Pull request by AhmedAbdoOrtiga**
#### Describe the change
- Rename Folders to Axe.Windows.* instead of AccessibilityInsights.*
- Replace "AccessibilityInsights" with "Axe.Windows" in all files in the repo:
  - .sln file.
  - .csproj files.
  - yml files.
  - .cs files like AssemblyInfo.cs.
  - .ps1 files like the automations checks.
  - .nuspec files.
  - .fakes files.
- Remove compliance yml file since it's not used anymore.
- Remove Axe.Windows prefix from Telemetry project.
- Alpha sort file paths in the signed build yml files.
- Remove check installed automation and release cleanup scripts since they're meant for AI-Win.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: build/compliance-build.yml [hunk 1]
```diff
@@ -105,14 +105,14 @@ jobs:
...
-                      src\\AccessibilityInsights.Automation\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Extensions\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Extensions.AzureDevOps\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Extensions.GitHub\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Extensions.GitHubAutoUpdate\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Extensions.Telemetry\\bin\\Debug\\*.dll"
+      AnalyzeTarget: "src\\Axe.Windows\\bin\\Debug\\*.exe;\
+                      src\\Axe.Windows\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Automation\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Extensions\\bin\\Debug\\*.dll;\
```
Comment (DaveTryon) [inline build/compliance-build.yml]: There are no Extensions in Axe.Windows...

File: build/compliance-build.yml [hunk 2]
```diff
@@ -125,14 +125,14 @@ jobs:
...
-                src\\AccessibilityInsights.Automation\\bin\\Debug\\AccessibilityInsights.*.dll;\
-                src\\AccessibilityInsights.Extensions\\bin\\Debug\\AccessibilityInsights.*.dll;\
-                src\\AccessibilityInsights.Extensions.AzureDevOps\\bin\\Debug\\AccessibilityInsights.*.dll;"\
-                src\\AccessibilityInsights.Extensions.GitHub\\bin\\Debug\\AccessibilityInsights.*.dll;\
-                src\\AccessibilityInsights.Extensions.GitHubAutoUpdate\\bin\\Debug\\AccessibilityInsights.*.dll;\
-                src\\AccessibilityInsights.Extensions.Telemetry\\bin\\Debug\\AccessibilityInsights.*.dll"
+      targets: "src\\Axe.Windows\\bin\\Debug\\*.exe;\
+                src\\Axe.Windows\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Automation\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Extensions\\bin\\Debug\\Axe.Windows.*.dll;\
```
Comment (DaveTryon) [inline build/compliance-build.yml]: Again, no Extensions...

File: src/Axe.Windows.Win32/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -26,27 +26,27 @@
...
 [assembly: InternalsVisibleTo("Axe.Windows.Core,PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
-[assembly: InternalsVisibleTo("AccessibilityInsights.DesktopUI,PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
+[assembly: InternalsVisibleTo("Axe.Windows.DesktopUI,PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
```
Comment (DaveTryon) [inline src/Axe.Windows.Win32/Properties/AssemblyInfo.cs]: DesktopUI is not part of Axe.Windows

File: src/Axe.Windows.Win32/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -26,27 +26,27 @@
...
-[assembly: InternalsVisibleTo("AccessibilityInsights.Extensions,PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
+[assembly: InternalsVisibleTo("Axe.Windows.Extensions,PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
 #else
-[assembly: InternalsVisibleTo("AccessibilityInsights")]
+[assembly: InternalsVisibleTo("Axe.Windows")]
 [assembly: InternalsVisibleTo("Axe.Windows.Desktop")]
 [assembly: InternalsVisibleTo("Axe.Windows.Core")]
-[assembly: InternalsVisibleTo("AccessibilityInsights.DesktopUI")]
+[assembly: InternalsVisibleTo("Axe.Windows.DesktopUI")]
```
Comment (DaveTryon) [inline src/Axe.Windows.Win32/Properties/AssemblyInfo.cs]: DesktopUI is not part of Axe.Windows

File: src/Axe.Windows.Fakes.Prebuild/Fakes.Prebuild.csproj [hunk 5]
```diff
@@ -7,8 +7,8 @@
     <ProjectGuid>{149576B1-9947-4BE0-9008-DFFD0DA36E00}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>AccessibilityInsights.Fakes.Prebuild</RootNamespace>
```
Comment (DaveTryon) [inline src/Axe.Windows.Fakes.Prebuild/Fakes.Prebuild.csproj]: Build fails if FAKES_SUPPORTED=1 is set in the environment. Not really sure why...

Comment (AhmedAbdoOrtiga) [reply src/Axe.Windows.Fakes.Prebuild/Fakes.Prebuild.csproj]: cause the fakes files weren't renamed too.

File: build/signedbuild.yml [hunk 6]
```diff
@@ -113,13 +113,13 @@ jobs:
...
-                      src\\AccessibilityInsights.RuleSelection\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Win32\\bin\\Debug\\*.dll;\
+      AnalyzeTarget: "src\\Axe.Windows.Actions\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Automation\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Core\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Desktop\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Rules\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.RuleSelection\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Win32\\bin\\Debug\\*.dll;\
                       src\\Axe.Windows.Telemetry\\bin\\Debug\\*.dll;"
```
Comment (DaveTryon) [inline build/signedbuild.yml]: These are intended to be alphabetically sorted, so Telemetry should come before Win32 (be sure to update the trailing backslashes)

File: build/signedbuild.yml [hunk 7]
```diff
@@ -133,13 +133,13 @@ jobs:
...
-                src\\AccessibilityInsights.RuleSelection\\bin\\Debug\\Axe.Windows.*.dll;\
-                src\\AccessibilityInsights.Win32\\bin\\Debug\\Axe.Windows.*.dll;\
+      targets: "src\\Axe.Windows.Actions\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Automation\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Core\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Desktop\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Rules\\bin\\Debug\\Axe.Windows.*.dll\
+                src\\Axe.Windows.RuleSelection\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Win32\\bin\\Debug\\Axe.Windows.*.dll;\
                 src\\Axe.Windows.Telemetry\\bin\\Debug\\Axe.Windows.*.dll;"
```
Comment (DaveTryon) [inline build/signedbuild.yml]: These are intended to be alphabetically sorted, so Telemetry should come before Win32 (be sure to update the trailing backslashes)

**Pull request merged**